uber/decorators.py

```diff
@@ -254,9 +254,7 @@ def with_session(*args, **kwargs):
         else:
             with sa.Session() as session:
                 try:
-                    retval = func(*args, session=session, **kwargs)
-                    session.expunge_all()
```

Comment:
This call to `expunge_all()` is not necessary, because the session is immediately closed anyway.